src/PowerShellEditorServices/Debugging/StackFrameDetails.cs

```diff
@@ -43,7 +63,7 @@ public class StackFrameDetails
         /// <summary>
         /// Gets the line number of the script where the stack frame occurred.
         /// </summary>
-        public int EndLineNumber { get; internal set; }
+        public int? EndLineNumber { get; internal set; }
```

Comment:
Considering that Get-PSCallStack doesn't return any info about end line/col it seems like these values should be nullable (even though I'm setting them to startLine / 0 respectively).